lib/active_shipping/carriers/correios.rb

```diff
@@ -21,8 +21,10 @@ def self.available_services
 
     protected
 
-    DEFAULT_SERVICES = [41106, 40010]
+    DEFAULT_SERVICES = ['04510', '04014']
```

Comment:
If we send `4510` instead of `04510` to the endpoint, it'll return an error saying the service is invalid.